js/hitbtc.js

```diff
@@ -499,7 +498,7 @@ module.exports = class hitbtc extends Exchange {
                 'precision': this.parseNumber (this.parsePrecision (precision)),
                 'limits': {
                     'amount': {
-                        'min': 1 / Math.pow (10, decimals),
+                        'min': this.parseNumber ('1e-' + precision),
```

Comment:
why not make use of `parsePrecision` here?

Reply:
I think it's supposed to be undefined actually. @kroitor can you confirm this?